### DISCUSSION TITLE
Mach3 board to T41U5XBB grblhal - servo drive connection

### DISCUSSION THREAD

**Discussion by Switch-Witch** [grblHAL/core#561] [Q&A]
O
<img width="173" alt="Bildschirmfoto 2024-07-31 um 14 09 07" src="https://github.com/user-attachments/assets/b8f315aa-0020-4ff2-88c1-72c0d3471129">
<img width="85" alt="Bildschirmfoto 2024-07-31 um 14 09 28" src="https://github.com/user-attachments/assets/2ad7e9f5-a6c6-4a37-acd8-9dba2ea04027">
n the way to replace my mach3 board with a T41U5XBB. On the mach3 board my servo drives were connected with dir- pulse- and a paired pulse/dir +. On the T41U5XBB the board offers one ground and dedicated + for pulse and dir. I tried to modify my cables that way but the servo drives are not getting a useable signal. When using the modified cables on a stepper driver it works without any problem. 
What to do to fix that problem?

**Comment by terjeio** [comment]
> What to do to fix that problem?

Either connect DIR+ and PUL+ to 5V or DIR- and PUL- to Gnd and DIR+ to Dir and PUL+ to Stp. Depending on your choice you may have to invert the step pulse ($2) and direction ($3) signals.

**Reply by Switch-Witch** [reply]
Thx a lot for your help.
The solution was the much higher resolution awaited from the servo encoder than a regular stepper motor.
Now they are moving - what to do to fine tune will be the next challenge 😄

Thank you
